### ISSUE TITLE
Very bad signal after installation

### ISSUE THREAD

**Issue by vyacheslav25** [kimocoder/realtek_rtwifi#18]
TP-Link TL-WN722N v2
Fedora 34, 5.11.14 
monitoring works well, but the signal of all points is terrible. Pings are normal, before you start doing anything with the network.
Tell me how to correctly remove this driver

**Comment by SisyphusIsntHappy** [comment]
I am experiencing the same issue. I'm not an expert but, this is probably the driver reporting false data unless there is an enumerator that the driver can control in the hardware.

**Comment by dubhater** [comment]
@SisyphusIsntHappy That's right. The driver is calculating the signal strength the same way for all the chips it supports, but that way is only correct for RTL8723AU and RTL8192CU families. This is now fixed upstream (rtl8xxxu driver in wireless-next).
